src/hotspot/cpu/aarch64/aarch64_vector_ad.m4

```diff
@@ -508,20 +541,21 @@ dnl
 // ------------------------------ Vector add -----------------------------------
 
 // vector add
-BINARY_OP(vaddB, AddVB, addv, sve_add,  B)
-BINARY_OP(vaddS, AddVS, addv, sve_add,  H)
-BINARY_OP(vaddI, AddVI, addv, sve_add,  S)
-BINARY_OP(vaddL, AddVL, addv, sve_add,  D)
-BINARY_OP(vaddF, AddVF, fadd, sve_fadd, S)
-BINARY_OP(vaddD, AddVD, fadd, sve_fadd, D)
+BINARY_OP(vaddB,  AddVB,  addv, sve_add,  B)
+BINARY_OP(vaddS,  AddVS,  addv, sve_add,  H)
+BINARY_OP(vaddI,  AddVI,  addv, sve_add,  S)
+BINARY_OP(vaddL,  AddVL,  addv, sve_add,  D)
+BINARY_OP(vaddHF, AddVHF, fadd, sve_fadd, H)
+BINARY_OP(vaddF,  AddVF,  fadd, sve_fadd, S)
+BINARY_OP(vaddD,  AddVD,  fadd, sve_fadd, D)
 
 // vector add - predicated
-BINARY_OP_PREDICATE(vaddB, AddVB, sve_add,  B)
-BINARY_OP_PREDICATE(vaddS, AddVS, sve_add,  H)
-BINARY_OP_PREDICATE(vaddI, AddVI, sve_add,  S)
-BINARY_OP_PREDICATE(vaddL, AddVL, sve_add,  D)
-BINARY_OP_PREDICATE(vaddF, AddVF, sve_fadd, S)
-BINARY_OP_PREDICATE(vaddD, AddVD, sve_fadd, D)
+BINARY_OP_PREDICATE(vaddB,  AddVB,  sve_add,  B)
```

Comment:
stylistic nit: remove the extra spaces.

In the initial commit, this extra space is added due to `vaddHF`. However, in the latest commit the predicated Float16 rules are removed.

Reply:
Done. Thanks!